apps/apps-cli/src/commands/install-app-command.ts

```diff
@@ -0,0 +1,35 @@
+import { installAndWaitForResult } from "../lib/install-and-wait-for-result";
+import { getAccessTokenMutation } from "../saleor-api/operations/get-access-token-mutation";
+import ora from "ora";
+
+interface InstallAppCommandArgs {
+  instanceUrl: string;
+  userEmail: string;
+  userPassword: string;
+  manifestUrl: string;
+}
+
+export const installAppCommand = async ({
+  instanceUrl,
+  manifestUrl,
+  userEmail,
+  userPassword,
+}: InstallAppCommandArgs) => {
+  const loginSpinner = ora("Logging into Saleor instance").start();
+
+  const token = await getAccessTokenMutation({
```

Comment:
AFAIK each time you log in, you invalidate previous token. Its not a big deal when you need to log in to dashboard again. But what if I want to install 10 apps? I will have to run CLI 10 times. Each next run will cause previous one to fail.

We need "batch install"  command to install multiple apps at once with single log in